example/ios/AdyenExampleTests/Modules/ApplePayModuleTests.swift

```diff
@@ -6,182 +6,210 @@
-  }
-
-  func testPKPaymentAuthorizationServiceAdapter() {
-      // Given
-    let adapter = PKPaymentAuthorizationServiceAdapter()
-
-      // When
-    let canMakePayments = adapter.canMakePayments(usingNetworks: [.visa, .masterCard])
-    let viewController = adapter.getAuthorizationViewController(paymentRequest: mockPaymentRequest)
-
-      // Then
-      // These methods delegate to the real PKPaymentAuthorizationViewController
-      // so we can't easily test the exact behavior without mocking the static methods
-      // But we can at least test that the methods don't crash and return expected types
-    XCTAssertNotNil(viewController)
-    XCTAssertTrue(canMakePayments)
-  }
-
-  func testHide() {
-      // Given
-    let expectation = self.expectation(description: "dismiss should be called")
-    let testModule = TestableApplePayModule(pkPaymentAuthorizationService: mockPaymentAuthorizationService)
-
-      // When
-    testModule.hide(NSNumber(value: true), event: NSDictionary())
-
-      // Then
-    DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
-      expectation.fulfill()
+
+    func test_pkContact_providesJsonObject() {
+        // GIVEN
+        let contact = PKContact()
+        contact.emailAddress = "test@example.com"
+        contact.phoneNumber = CNPhoneNumber(stringValue: "+1234567890")
+
+        var name = PersonNameComponents()
+        name.givenName = "John"
+        name.familyName = "Doe"
+        contact.name = name
+
+        // WHEN
+        let jsonObject = contact.jsonObject
+
+        // THEN
+        XCTAssertNotNil(jsonObject)
+    }
+
+    func test_pkPaymentNetwork_hasTxVariantName() {
+        // GIVEN
+        let networks = [
+            PKPaymentNetwork.masterCard,
+            PKPaymentNetwork.cartesBancaires,
+            PKPaymentNetwork.visa,
+            PKPaymentNetwork.amex
+        ]
+        
+        // WHEN/THEN
+        XCTAssertEqual(networks[0].txVariantName, "mc")
+        XCTAssertEqual(networks[1].txVariantName, "cartebancaire")
+        XCTAssertEqual(networks[2].txVariantName, "visa")
+        XCTAssertEqual(networks[3].txVariantName, "amex")
+    }
+
+    func test_pkPaymentAuthorizationServiceAdapter_providesPaymentServices() {
+        // GIVEN
+        let adapter = PKPaymentAuthorizationServiceAdapter()
+
+        // WHEN
+        let canMakePayments = adapter.canMakePayments(usingNetworks: [.visa, .masterCard])
+        let viewController = adapter.getAuthorizationViewController(paymentRequest: mockPaymentRequest)
+
+        // THEN
+        // These methods delegate to the real PKPaymentAuthorizationViewController
+        // so we can't easily test the exact behavior without mocking the static methods
+        // But we can at least test that the methods don't crash and return expected types
+        XCTAssertNotNil(viewController)
+        XCTAssertTrue(canMakePayments)
+    }
+
+    func test_hide_callsDismiss() {
```

Comment:
This one still succeeds no matter what happens inside hide, right?
Expectation is always fulfilled after 0.1 timeout

Reply:
Good spot!

Reply:
Addressed in [0877f04](https://github.com/Adyen/adyen-react-native/pull/717/commits/0877f04deb6cbaa5c169ed396d988fcffab743ba)